.github/workflows/dotnet-core-linux.yml

```diff
@@ -3,6 +3,7 @@ name: .NET Linux
 on:
   pull_request:
     branches: [ master ]
+    - '**.md'
```

Comment:
I think we have to change to:
```
paths-ignore:
- '**/**.md'
```

Reply:
That doesn't seem correct to me. Take a look at this https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-paths.